frontend/public/redux.ts

```diff
@@ -40,6 +40,9 @@ const store = createStore(
   composeEnhancers(applyMiddleware(thunk)),
 );
 
+// Make store globally available for dynamic plugins
```

Comment:
Isn't this what `storeHandler.getStore()` is for?

Reply:
@Leo6Leo @TheRealJon

The root cause of the problem seems to be that `storeHandler.setStore` function is not called right after the Redux store instance is created and initialized in `public/redux.ts` module.

So I think the only change we need for this bugfix is to modify `public/redux.ts` like so:
```ts
// ...
import storeHandler from '@console/dynamic-plugin-sdk/src/app/storeHandler';
// ...

storeHandler.setStore(store);

export default store;
```

Bugfix PRs should contain minimal changes to address the immediate issue. We can further improve the Redux handling code in follow-up non-bugfix PRs.